editor.planx.uk/src/pages/FlowEditor/lib/store/editor.ts

```diff
@@ -83,6 +86,9 @@ export const editorUIStore: StateCreator<
 
     hideTestEnvBanner: () => set({ isTestEnvBannerVisible: false }),
 
+    isLive: isLivePlatform,
+    isTest: isTestPlatform,
```

Comment:
I would say it make more sense to put the logic here for controlling / setting the environment rather than in the theme 👍